src/app/components/user/UserConnectedAccount.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Relay from 'react-relay/classic';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { ListItem } from 'material-ui/List';
+import FlatButton from 'material-ui/FlatButton';
```

Comment:
Would be preferable to use `@material-ui/core` components as we are phasing out the older `material-ui` version.

Reply:
Done